loadtest/utils.go

```diff
@@ -87,3 +89,11 @@ func nextPowerOf2(val int) int {
 	val++
 	return val
 }
+
+func RandomFutureTime(deltaStart, maxUntil time.Duration) int64 {
+	now := time.Now()
+	diff := maxUntil - deltaStart
+	offset := deltaStart.Milliseconds() + rand.Int63n(diff.Milliseconds())
+	randomFutureTime := now.Add(time.Duration(offset) * time.Millisecond)
+	return randomFutureTime.UnixMilli()
+}
```

Comment:
Let's add a small comment here (I think I got the interval right, but please review it yourself, it's confusing :joy:):
```suggestion
// RandomFutureTime returns a random Unix timestamp, in milliseconds, in the interval
// [now+deltaStart, now+deltaStart+maxUntil]
func RandomFutureTime(deltaStart, maxUntil time.Duration) int64 {
	now := time.Now()
	diff := maxUntil - deltaStart
	offset := deltaStart.Milliseconds() + rand.Int63n(diff.Milliseconds())
	randomFutureTime := now.Add(time.Duration(offset) * time.Millisecond)
	return randomFutureTime.UnixMilli()
}
```

Reply:
Done, updated.